applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.cpp

```diff
@@ -935,13 +935,13 @@ void UPwSmallStrainFICElement<2, 4>::CalculateAndAddStrainGradientMatrix(MatrixT
 {
     KRATOS_TRY
 
-    noalias(rVariables.PUMatrix) =
+    BoundedMatrix<double, 4, 4 * 2> strain_gradient_matrix =
```

Comment:
Can't we use `const auto` here?
```suggestion
    const auto strain_gradient_matrix =
```

To avoid having many similar comments, could you please check your other changes as well? Thank you.

Reply:
added `const` but not `auto`. In similar using `const auto` caused an error. Added many `const`s